pkg/kubelet/dockershim/docker_service.go

```diff
@@ -412,7 +412,8 @@ func (ds *dockerService) Start() error {
 
 	go func() {
 		if err := ds.streamingServer.Start(true); err != nil {
-			klog.Fatalf("Streaming server stopped unexpectedly: %v", err)
+			klog.ErrorS(err, "Streaming server stopped unexpectedly")
+			os.Exit(255)
```

Comment:
Migration guide proposes to change status code to 1
```suggestion
			os.Exit(1)
```